Bottom: 67a632edad5dd3fdbbd06a6f851910e7cda522ef
Top:    72e9836ee755017d3d5e2a4e2401c1d1560728bb
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-27 15:04:35 +0100

fontmode.c: Add missing include


---

diff --git a/src/fontmode.c b/src/fontmode.c
index 985a38f..dd64f63 100644
--- a/src/fontmode.c
+++ b/src/fontmode.c
@@ -46,6 +46,7 @@
 #include "rtree.h"
 #include "strflags.h"
 #include "undo.h"
+#include "pcb-printf.h"
 
 #ifdef HAVE_LIBDMALLOC
 #include <dmalloc.h>
